storm-server/src/main/java/org/apache/storm/scheduler/DefaultScheduler.java

```diff
@@ -101,6 +101,10 @@ public void prepare(Map<String, Object> conf) {
         //noop
     }
 
+    @Override
+    public void cleanup() {
+    }
```

Comment:
Most of the cleanup() method seems a no-op, can we define a default implementation of it ?

Reply:
@danny0405 - Updated.  Please take a look.